templates/template-active-vendor-dashboard.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Activate dokan vendor dashboard notice.
+ */
+if ( ! defined( 'ABSPATH' ) ) {
+    exit;
+}
+?>
+<div class="notice dokan-admin-notices-wrap dokan-migrator-active-v-dash-notice">
+    <div class="dokan-admin-notice dokan-alert">
+        <div class="notice-content">
+            <div class="logo-wrap">
+                <div class="dokan-logo"></div>
+                <span class="dokan-icon dokan-icon-alert"></span>
+            </div>
+            <div class="dokan-message">
+                <h3><?php esc_html_e( 'Activate dokan vendor dashboard.', 'dokan-migrator' ); ?></h3>
+                <?php
+                    // translators: 1$-2$: opening and closing a tag to active dokan vendor dashboard.
+                    $text = sprintf( esc_html__( '%1$sActive now%2$s', 'dokan-migrator' ), '<a id="dokan-migrator-active-v-dash" href="#">', '</a>' );
+
+                ?>
+                <div><?php echo $text; ?></div>
+            </div>
+        </div>
+    </div>
+</div>
+
+<script>
+    ;(function($){
```

Comment:
Can this portion of script be moved to the main script so that it can be executed from there?